src/main/components/Header.js

```diff
@@ -94,9 +95,49 @@ const Header = observer(() => {
             Discover Projects
           </Action>
         </nav>
-        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => {}} />
+        <Action icon={MenuIcon} className="dark header__mobile-nav-button mobile" onClick={() => setShowMobileMenu(prevState => !prevState)} />
       </header>
 
+      <div className={`mobile-nav__menu ${showMobileMenu ? "" : "hidden"}`}>
```

Comment:
Let's move the mobile menu to a different file, and don't actually render it unless it's open.